README.md

```diff
@@ -16,7 +16,7 @@ OpenEMR is a leader in healthcare open source software. Costly proprietary EMRs
 
 ### Support
 
-Community and Professional support can be found [here](https://open-emr.org/wiki/index.php/OpenEMR_Support_Guide).
+Professional support can be found [here](https://open-emr.org/wiki/index.php/OpenEMR_Support_Guide).
 
```

Comment:
hi @sanjeevkumarray , doesn't this link also show the community option?
![Screenshot from 2021-01-31 17-36-41](https://user-images.githubusercontent.com/5618372/106400171-e9be6c00-63ea-11eb-9e21-89135eee1fe9.png)

Reply:
@stephenwaite He's just spamming the repo with junk PRs. Some so-called influencer on youtube is teaching people to do these kinds of spammy PRs.